assets/js/modules/analytics-4/index.js

```diff
@@ -690,33 +696,96 @@ export const registerWidgets = ( widgets ) => {
 	);
 };
 
+export const ANALYTICS_4_NOTIFICATIONS = {
+	[ AUDIENCE_SEGMENTATION_SETUP_SUCCESS_NOTIFICATION ]: {
+		Component: AudienceSegmentationSetupSuccessSubtleNotification,
+		priority: 10,
+		areaSlug: NOTIFICATION_AREAS.BANNERS_BELOW_NAV,
+		viewContexts: [ VIEW_CONTEXT_MAIN_DASHBOARD ],
+		checkRequirements: async ( { select, resolveSelect } ) => {
+			const analyticsConnected = await resolveSelect(
+				CORE_MODULES
+			).isModuleConnected( 'analytics-4' );
+
+			if (
+				! analyticsConnected ||
+				! isFeatureEnabled( 'audienceSegmentation' )
+			) {
+				return false;
+			}
+
+			await resolveSelect( MODULES_ANALYTICS_4 ).getSettings();
+			const configuredAudiences =
+				select( CORE_USER ).getConfiguredAudiences();
+
+			// Only show this notification if the user has a set of configured audiences.
+			return Array.isArray( configuredAudiences );
+		},
+		isDismissible: true,
+	},
+	[ AUDIENCE_SEGMENTATION_SETUP_CTA_NOTIFICATION ]: {
+		Component: AudienceSegmentationSetupCTAWidget,
+		priority: 10,
+		areaSlug: NOTIFICATION_AREAS.BANNERS_BELOW_NAV,
+		groupID: NOTIFICATION_GROUPS.SETUP_CTAS,
+		viewContexts: [ VIEW_CONTEXT_MAIN_DASHBOARD ],
+		checkRequirements: async ( { select, resolveSelect } ) => {
+			const analyticsConnected = await resolveSelect(
+				CORE_MODULES
+			).isModuleConnected( 'analytics-4' );
+
+			if ( ! analyticsConnected ) {
+				return false;
+			}
+
+			await Promise.all( [
+				resolveSelect( CORE_MODULES ).isModuleConnected(
+					'analytics-4'
+				),
+				resolveSelect( CORE_USER ).getDismissedPrompts(),
+				select( CORE_USER ).getAudienceSettings(),
+				select( MODULES_ANALYTICS_4 ).isGatheringData(),
+				resolveSelect( MODULES_ANALYTICS_4 ).getSettings(),
+			] );
+
+			const isDismissed = select( CORE_USER ).isPromptDismissed(
+				AUDIENCE_SEGMENTATION_SETUP_CTA_NOTIFICATION
+			);
+
+			const configuredAudiences =
+				select( CORE_USER ).getConfiguredAudiences();
+
+			const analyticsIsDataAvailableOnLoad =
+				select( MODULES_ANALYTICS_4 ).isDataAvailableOnLoad();
+
+			const audienceSegmentationSetupCompletedBy =
+				select(
+					MODULES_ANALYTICS_4
+				).getAudienceSegmentationSetupCompletedBy();
+
+			if (
+				audienceSegmentationSetupCompletedBy !== null ||
+				configuredAudiences === undefined ||
+				configuredAudiences?.length ||
+				! analyticsIsDataAvailableOnLoad ||
+				isDismissed ||
+				! isFeatureEnabled( 'audienceSegmentation' )
```

Comment:
Let's check for this earlier, in case any of these selectors aren't available.

(And also: to avoid running any resolvers needlessly.)